### PR TITLE
2.1 logging description

### PR DESCRIPTION
**Pull request by bbrks**
Adds documentation about 2.1 logging

### CODE REVIEW

File: md-docs/_20/guides/sync-gateway/logging.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+This allows system administrators running Sync Gateway to tweak the log level, and log keys for the console output to suit their needs, whilst maintaining the level of logging required by Couchbase Support for investigation of issues.
+
+## Console Log Output
+
+The console output of Sync Gateway can be filtered down via log levels and log keys, and you can tweak this as much as you like without impacting Support's ability to analyze the log files described in [Log File Outputs](index.html#log-file-outputs).
+
+### Log Levels
+
+The console log output can be configured with the following log levels, ordered from least verbose, to most. Note that log levels are additive, so if you enable `info` level, `warn` and `error` logs are also enabled.
```
Comment (jamiltz) [inline md-docs/_20/guides/sync-gateway/logging.md]: You should call out the default log level (if any?).

Comment (jamiltz) [reply md-docs/_20/guides/sync-gateway/logging.md]: Perhaps it's the same as log file defaults https://github.com/couchbaselabs/couchbase-mobile-portal/pull/869/files#diff-5ec6deaaa8d7d18a32f41e30258d78d0R60? If so I think we should add a column to the Log Levels table called "Enabled by default" or something like that.

**Comment by bbrks** [comment]
Thanks @jamiltz, I've updated with your feedback, and fixed the conflict on the nav map.

**Comment by jamiltz** [comment]
github doesn't let me rebase and merge

image

could be because of the merge commit - will try to rebase locally and push again.

**Pull request merged**